package.json

```diff
@@ -7,21 +7,21 @@
   "scripts": {
     "prebuild": "npm run build --workspace=@amsterdam/asc-assets",
     "build": "npm run clean:generated && npm run build:ts && npm run build:ts:es && npm run build:ts:beta",
-    "build:watch": "npm run clean:generated && concurrently \"npm run build:ts -- --watch\" \"npm run build:ts:es -- --watch\"",
-    "build:watch:beta": "npm run clean:generated && npm run build:ts:beta -- --watch",
+    "build:watch": "npm run clean:generated && concurrently \"npm run build:ts -- --watch\" \"npm run build:ts:es -- --watch\" \"npm run build:ts:beta\"",
+    "build:beta:watch": "npm run clean:generated && concurrently \"npm run build:ts\" \"npm run build:ts:es\" \"npm run build:ts:beta  -- --watch\"",
```

Comment:
Logical change, but not in-line with the new script commands below

Reply:
Do you mean the order of keywords? Maybe `build:ts:beta` should be `build:beta`, seeing as it build both `ts` and `es`